doc/man/zend.1

```diff
@@ -76,7 +76,7 @@ mempool will accept (default: 0 = no limit applied)
 .HP
 \fB\-par=\fR<n>
 .IP
-Set the number of script verification threads (\fB\-8\fR to 16, 0 = auto, <0 =
+Set the number of script verification threads (\fB\-4\fR to 16, 0 = auto, <0 =
```

Comment:
It seems to me the GetNumCores within util.cpp is system-dependent; btw, I have manually updated zend.1 so that this line is equal to previous version.
Addressed in 7e9b6c349cfa36938572fc1ebc6c0426139e48cd.